### PR TITLE
Enable CI

### PR DESCRIPTION
**Pull request by wangkuiyi**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -156,6 +156,5 @@ RUN mkdir -p /go/bin
 ENV GOPATH /go
 ENV PATH $PATH:$GOPATH/bin
 
-# Install Go's dependency management tool.
-RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-
+RUN echo "go get -t sqlflow.org/gohive && go test -v sqlflow.org/gohive" > /build_and_test.bash
```
Comment (tonyyang-svail) [inline docker/Dockerfile:159]: `go get -t sqlflow.org/gohive` will download and test the develop branch. Where is the test on the pull request branch?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+branches:
+  only:
+  - gh-pages
+  - /.*/
+
+service:
+  - docker
+
+script:
+  - docker run --rm -it sqlflow/gohive:dev /build_and_test.bash
```
Comment (tonyyang-svail) [inline .travis.yml]: Should we mount the pull request branch into the Docker container?

Comment (wangkuiyi) [reply .travis.yml]: Oh. my god. Yes, I am a fool

**Comment by wangkuiyi** [comment]
This PR wouldn't pass CI before we merge https://github.com/sql-machine-learning/gohive/pull/11 and DockerHub automatically rebuild the image `sqlflow/gohive:dev`.

**Pull request merged**